doc-content/shared-kie-docs/src/main/asciidoc/KieServer/securing-passwords-proc.adoc

```diff
@@ -0,0 +1,95 @@
+[id='securing-passwords-proc_{context}']
+
+= Securing {KIE_SERVER} passwords with a keystore
+
+{KIE_SERVER} uses passwords to authenticate users. For increased security, you can store {KIE_SERVER} passwords in a keystore. Because it supports symmetrics keys, you must use a Java Cryptography Extension KeyStore (JCEKS) with {KIE_SERVER}. Use KeyTool, which is part of the JDK installation, to create a new JCEKS.
```

Comment:
To clarify this a bit - the keystore security is not dedicated just to Kie server, it is meant to encrypt passwords used for communication between Workbench and Kie server - both controller password and Kie server password should be encrypted.
In case Workbench and Kie server are deployed to different application servers then both application servers should use the keystore.